app/knowledge_panels.py

```diff
@@ -39,3 +44,43 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def last_edits_kp(facet: str, value: str):
+    if facet == "packaging":
+        plural_facet = facet
+        # That how the api read packaging
+    else:
+        plural_facet = p.plural(facet)
+    """
+    Changing other facet to plural because that how the search api works
+    """
+    query = {}
+    if value is not None:
+        query[f"{plural_facet}_tags_en"] = value
+        query["fields"] = "code,product_name,last_editor"
+        query["sort_by"] = "last_modified_t"
+    search_url = "https://world.openfoodfacts.org/api/v2/search"
+    if query:
+        search_url += f"?{urlencode(query)}"
+
+    response_API = requests.get(search_url)
+    data = response_API.text
+    parse_json = json.loads(data)
+    counts = parse_json["count"]
+    tags = parse_json["products"]
```

Comment:
There again requests helps you:
```suggestion
    data = response_API.json()
    counts = data["count"]
    tags = data["products"]
```